streamalert_cli/terraform/rules_engine.py

```diff
@@ -42,6 +42,9 @@ def generate_rules_engine(config):
         'threat_intel_enabled': config.get('threat_intel', {}).get('enabled'),
         'dynamodb_table_name': config.get('threat_intel', {}).get('dynamodb_table_name'),
         'rules_table_arn': '${module.globals.rules_table_arn}',
+        'enable_rule_staging': config['global']['infrastructure']['rule_staging'].get(
```

Comment:
For naming consistency, I would use `rule_staging_enabled`. See `threat_intel_enabled` naming at line 42.

Reply:
I hate to contradict you here, but I actually like the convention of `enable_*` better.. I don't like the tense-y version of `*_enabled` much

Reply:
by tense-y I mean past test like enable**d**

Reply:
![cat_bff](https://user-images.githubusercontent.com/21151436/72492731-22d17180-37d3-11ea-9c15-ab604cefdd45.gif)

Reply:
This is a copy of a variable that already exists in the terraform modules. I could change it but then I need to change it across the board